tfx/components/example_validator/component_test.py

```diff
@@ -30,11 +30,12 @@ class ExampleValidatorTest(tf.test.TestCase):
   def testConstruct(self):
     statistics_artifact = standard_artifacts.ExampleStatistics()
     statistics_artifact.split_names = artifact_utils.encode_split_names(
-        ['eval'])
+        ['train', 'eval'])
+    exclude_splits = ['train']
     example_validator = component.ExampleValidator(
         statistics=channel_utils.as_channel([statistics_artifact]),
         schema=channel_utils.as_channel([standard_artifacts.Schema()]),
-    )
+        exclude_splits=exclude_splits)
```

Comment:
check if example_validator.exec_properties contains the correct value, same elsewhere